### ISSUE TITLE
Cooldown

### ISSUE THREAD

**Issue by aaron13ps** [x3chaos/RTD#3]
The Cool-Down does not affect when able to use the command.
In other words, the cool-down does not work

**Comment by x3chaos** [comment]
Is your cooldown set?  
EG:

```
options:
    cooldown: 30
```

**Comment by aaron13ps** [comment]
Yes, it is set for 60 at the moment, but I can perform RTD whenever I want to (even as a guest).

options:
    cooldown: 60

Off Topic suggestions:
   -PLEASE add the permission, would be extremely help-full
   -Maybe have 2 types of player imputes, one being {player} (regular players name) and {dplayer} (display name of player)

**Comment by x3chaos** [comment]
I'm in full rewrite mode as we speak; it was bugging out badly because it was thrown together haphazardly.
